object_service/tests/test_queries.py

```diff
@@ -56,9 +57,9 @@ def test_query_translation(self):
             'citations(object:(X OR Y) OR fulltext:X) -foo:bar object:Z':'citations(((=abs:(X OR Y) OR simbid:(123 OR 456) OR nedid:(XXX OR YYY)) database:astronomy) OR fulltext:X) -foo:bar ((=abs:Z OR simbid:789 OR nedid:0) database:astronomy)'
         }
         
-        for qstring, expected in test_cases.items():
+        for qstring, expected in list(test_cases.items()):
```

Comment:
I think this `list(...)` (automatically added by the porting tools) might not be needed here.